products/opensuse/main.pm

```diff
@@ -391,6 +391,7 @@ sub load_consoletests() {
             loadtest "console/syslinux.pm";
         }
         loadtest "console/mtab.pm";
+        loadtest "console/rails.pm";
```

Comment:
this would break stagings atm as the rails stack is not in rings. So possibly we start with this as extra test